pyproj/_datadir.pyx

```diff
@@ -51,3 +51,19 @@ cdef void pyproj_context_initialize(
 def pyproj_global_context_initialize():
     proj_log_func(NULL, NULL, pyproj_log_function)
     set_context_data_dir(NULL)
+
+
+def get_user_data_dir(bint create: bool = True) -> str:
+    """
+    Parameters
```

Comment:
Can you indicate here in the docstring how this is different from `get_data_dir` ?

Reply:
Good thinking. Added some information.